resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager

```diff
@@ -1,2 +1,18 @@
+#
```

Comment:
Doesn't seem like other META-INF directories under spark-sql or ml-lib have this header.